pkg/apis/cert/v1alpha1/issuer.go

```diff
@@ -69,6 +69,31 @@ type ACMESpec struct {
 	// PrivateKeySecretRef is the secret ref to the ACME private key.
 	// +optional
 	PrivateKeySecretRef *corev1.SecretReference `json:"privateKeySecretRef,omitempty"`
+
+	// ACMEExternalAccountBinding is a reference to a CA external account of the ACME server.
+	// +optional
+	ExternalAccountBinding *ACMEExternalAccountBinding `json:"externalAccountBinding,omitempty"`
+
+	// NoDNSEntryCreation is a flag to disable creating of DNS entries/records for a DNS Challenge.
+	// For some setups, e.g. when using ExternalAccountBinding with a SAP DigiCert account, the DNS challenges are only performed pro forma.
+	// +optional
+	NoDNSEntryCreation *bool `json:"noDNSEntryCreation,omitempty"`
+
+	// DomainRanges allows to restrict the certificate requests to subdomains of the given domain names
+	// +optional
+	DomainRanges []string `json:"domainRanges,omitempty"`
```

Comment:
Do you think it makes sense to work with include and exclude domains like the [DNS controller](https://github.com/gardener/external-dns-management)?

```yaml
domains:
  include: []
  exclude: []
```

Reply:
I don't think it is a good fit here. In the DNS controller the situation is more complicated. The provider reports the base domains and forwarded domains of the hosted zone. The latter must be excluded.
Here a positive list should be sufficient.

Reply:
> The latter must be excluded.
> Here a positive list should be sufficient.

Can you make an example?

The reason I brought it to the table is that the `DomainRanges` seem very open as they automatically include every sub-domain and I'm already hearing users asking for exceptions to limit the scope. This again will lead to an awkward API structure as we needed to add it afterwards.

Reply:
If you already see a concrete need for it, let's do it as suggested by you. You are right, that it would be awkward to change it later. Thanks for bringing this up.

Reply:
done